### ISSUE TITLE
Notification - Mute by default some spaces 

### ISSUE THREAD

**Issue by Julien-Dubois-eXo** [Meeds-io/meeds#1316]
**Is your feature request related to a problem? Please describe.**
It's frequent we have some spaces that are open to all users and that are dedicated to training or examples. Every user can use it to test the platform and test some features. The drawback is all the members are notified of trash activities. Frequently it's new users that are concerned so they are not aware of the mute feature and how to do it.  

**Describe the solution you'd like**
It might be interesting to allow to mute by default a space. 
1st proposition: it could be in the admin menu, we set by default this space as mute and the members can unmute it if they want
2n proposition: in the space parameter, the host can set that the space is by default muted and the members can unmute it if they want.

I prefer the first option as the host may not understand the impact and the user will not be notified. And the needs are more linked to a space managed by an admin. 

**Describe alternatives you've considered**

**Additional context**

**Comment by srenault-meeds** [comment]
> 1st proposition: it could be in the admin menu, we set by default this space as mute and the members can unmute it if they want

IMO, rejected as it is a user decision, not an admin

> 2n proposition: in the space parameter, the host can set that the space is by default muted and the members can unmute it if they want.

IMO, rejected. In that case, if the host thinks the space is not useful, then don't make it mandatory

**Comment by Julien-Dubois-eXo** [comment]
@srenault-meeds I do not agree. We provide a poor experience with pre-settings notifications. I have frequent customer requests to help the user pre-set notification choices according to their context. They want to mute by default some specific notifications and let the user enable them afterward if they want. 

Pre-mute a space is one of these pre-settings. In some cases, the space is not used for collaboration or communication. They are used only for training and the notification is not necessary for the space, it's the opposite. The only way we have is to tell them to mute it. 

> In that case, if the host thinks the space is not useful, then don't make it mandatory
For a sandbox/training space, we may want the users to be by default members but can leave. In that case, the notifications are a drawback. Users will have more of the reflex to leave it instead of mute it. So as we can't mute it the customer doesn't want to add it by default to the users in these spaces even if they can leave it. The training spaces are not really used because they are not enough visible and the user have to think where to find it and shortcut is not enough.

By the way, I don't find encouraging to leave improvement feedback if they are just rejected without any discussion possibility.

**Comment by srenault-meeds** [comment]
ok let's discuss it then

**Comment by Hopeosim0901** [comment]
Would love to know more
